From 4fcf87b430ce5c06b582f5385404789931e1453c Mon Sep 17 00:00:00 2001
From: afl1 <afl2001@gmail.com>
Date: Wed, 7 Aug 2019 04:00:16 +0100
Subject: [PATCH 12/43] Fix missing fps hints.

Use 29.97 fps for missing fps hints.

Patch: kodi-le-039-amlcodec-missing-fps-hints.patch
---
 xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
index b62837b..45e7479 100644
--- a/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
+++ b/xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp
@@ -1557,6 +1557,8 @@ bool CAMLCodec::OpenDecoder(CDVDStreamInfo &hints)
     // then ffmpeg avg_frame_rate next
     am_private->video_rate = 0.5 + (float)UNIT_FREQ * hints.fpsscale / hints.fpsrate;
   }
+  else
+    am_private->video_rate = 0.5 + (float)UNIT_FREQ * 1001 / 30000;
 
   // check for 1920x1080, interlaced, 25 fps
   // incorrectly reported as 50 fps (yes, video_rate == 1920)
-- 
2.7.4

